props/prop_base.h

```diff
@@ -293,6 +297,36 @@ class PropBase : CommandParser, Looper, protected SaberBase {
 #endif    
   }
 
+  void ResumePreset() {
+    FileReader f;
+    int newPreset;
+    int newVolume;
+    if (f.Open("savedpreset.ini") && f.FileSize() > 0) {
+      for (; f.Available(); f.skipline()) {
+        char variable[33];
+        f.readVariable(variable);
+        if (!variable[0]) continue;
+        if (f.Peek() != '=') continue;
+        f.Read();
+        f.skipwhite();
+        if (!strcmp(variable, "preset")) {
+          newPreset = f.readIntValue();
+        continue;
+        }
+        if (!strcmp(variable, "volume")) {
+          newVolume = f.readIntValue();
+        }
+      }
+      f.Close();
+      SetPreset(newPreset, false);
```

Comment:
So does this mean that we will save the preset right after reading it from the file?
If so, that means a lot of extra writing, which can wear out the SD card.

Reply:
we need to save the state each time the font is changed.  I've moved the savestate (renamed savepreset) function to the next_ and prev_preset functions in prop_base.h, so I don't think i'm doing more than the required read/writes to make this work.